extensions/amp-story/1.0/variable-service.js

```diff
@@ -73,9 +73,9 @@ export class AmpStoryVariableService {
       [AnalyticsVariable.BOOKEND_COMPONENT_POSITION]: null,
       [AnalyticsVariable.BOOKEND_COMPONENT_TYPE]: null,
       [AnalyticsVariable.BOOKEND_TARGET_HREF]: null,
-      [AnalyticsVariable.INTERACTIVE_ELEMENT_ID]: null,
-      [AnalyticsVariable.INTERACTIVE_ELEMENT_RESPONSE]: null,
-      [AnalyticsVariable.INTERACTIVE_ELEMENT_TYPE]: null,
+      [AnalyticsVariable.STORY_REACTION_ELEMENT_ID]: null,
```

Comment:
Nit: do you need to keep the `_ELEMENT_` part?

Reply:
Good catch, I missed this one part when updating!